.env.alfajores

```diff
@@ -50,10 +50,10 @@ AZURE_ORACLE_CENTRALUS_FULL_NODES_DISK_SIZE=30
 
 # Temporarily point to celo-org repository to consume patched image.
 GETH_NODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-org/geth"
-GETH_NODE_DOCKER_IMAGE_TAG="1.1.0"
+GETH_NODE_DOCKER_IMAGE_TAG="1.2.2"
 
 GETH_BOOTNODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/geth-all"
-GETH_BOOTNODE_DOCKER_IMAGE_TAG="990a7a2b42ac8edbfb7389687c350baa657f583e"
+GETH_BOOTNODE_DOCKER_IMAGE_TAG="master"
```

Comment:
I don't think `celo-testnet` has the version docker tags. +1 to pinning the version though.

Reply:
I didn't notice it was celo-testnet rather than celo-org.  Is there a reason for that?  As far as I can tell, we also have geth-all on celo-org.

Reply:
The bootnode healthcheck does not work in version `1.2.2`. At least when I was trying it needed the [latest commits from master](https://github.com/celo-org/celo-blockchain/commits/68e2ab747b40ce215a916cde7fdc77613366dcd0/cmd/devp2p/discv4cmd.go).

Reply:
Oh, I see.  Didn't think of that, but it makes sense, because the `--networkid` flag for `discv4` was fixed in https://github.com/celo-org/celo-blockchain/pull/1212, which was after the v1.2.x branch cut.  We should update this to 1.3.0 when the time comes.